configure.bat

```diff
@@ -28,20 +28,6 @@ exit /B %errorlevel%
     )
     set envfile=
 
-    set all_set=1
-    if not defined OPENSSL_DIR set all_set=0
-
-    if %all_set%==1 (
-        @echo - Using OpenSSL: %OPENSSL_DIR%
-    ) else (
-        @echo At least one of the following variables were not set:
-        @echo     - OPENSSL_DIR: %OPENSSL_DIR%
-        @echo Please define them using by creating a "env.bat" file containing:
-        @echo     @set OPENSSL_DIR=^<your OpenSSL directory^>
-        exit /B 1
-    )
-    set all_set=
-
```

Comment:
If we remove this completely, ones that don't have `openssl_dir` defined will never understand the problem.
We should maintain this as the `else` of `if defined openssl_dir`.

Reply:
But the "problem" does not exist anymore. We don't need to enforce `openssl_dir` by any means. If openssl is not going to be found for someone, meson will just not find it and then we'll no the problem. Letting this in a guide would be better than in the script, I think.

What I can do, tho, is...

Reply:
[this](https://github.com/expertisesolutions/efl/compare/890e774274e053c26dd821bb7f5941eb563bdcec..df233053e17d2a93ea39107fc73a777fdb90cad6).